reference/admin.yaml

```diff
@@ -362,40 +362,46 @@ components:
         is_enabled:
           $ref: '#/components/schemas/is_enabled'
         restricted_spender_user_ids:
-          $ref: '#/components/schemas/restricted_spender_user_ids'
-        org_id:
-          $ref: '#/components/schemas/org_id'
-    allow_users:
-      type: string
-      nullable: false
-      description: >
-        Set who can access the object.
-
-        - `ALL` indicates the object can be accessible to all users in the org.
-
-        - `NONE` indicates the object cannot be accessible to any user in the
-        org.
-
-        - `UNRESTRICTED` indicates the object will be be accessible to all users
-        in the org who are not restricted to
+          type: array
+          nullable: true
+          items:
+            type: string
+            nullable: false
+            maxLength: 15
+            description: |
+              This id provided by Fyle to identify an object.
+            example: sdfd2391
+          example:
+            - uswoirwlwwg
+            - uswlgwkgw42
+          description: >
+            - List of IDs of users who can access this object.
 
-        specific objects of this resource.
-      enum:
-        - ALL
-        - NONE
-        - UNRESTRICTED
-      example: UNRESTRICTED
-    cost_center_restrict_in:
-      type: object
-      required:
-        - id
-        - allow_users
-      additionalProperties: false
-      properties:
-        id:
-          $ref: '#/components/schemas/id_integer'
-        allow_users:
-          $ref: '#/components/schemas/allow_users'
+            - In the same request payload, only one of
+            restricted_spender_user_ids or restricted_spender_user_group key
+              can be used to set restrictions.
+        restricted_spender_user_group:
+          type: object
+          required:
+            - with_no_cost_center_restrictions
+          additionalProperties: false
+          properties:
+            with_no_cost_center_restrictions:
+              type: boolean
+              description: >
+                - If you want to allow the cost_center to be visible to all the
+                users in the org with cost center restrictions,
+                  you can this key to True.
+                - In above use case, it eliminates the need of finding the
+                user_ids and then adding it in the restricted_spender_user_ids.
+
+                - In the same request payload, only one of
```

Comment:
Note: Only one of `restricted_spender_user_ids` or `restricted_spender_user_group` key can be used to set restrictions.
Note: Either `restricted_spender_user_ids` or `restricted_spender_user_group` {key} can be used to set restrictions. They both can't be used together.
Note: When you are using this {key}, you cannot use `restricted_spender_user_group` {key}.
Note: You can only use either this {key} or `restricted_spender_user_group` {key} for setting restrictions.
Note: You cannot use both this {key} in combination of `restricted_spender_user_group` {key} for setting restrictions

Reply:
Done, choose point 2